stage1/rootfs/usr_from_src/Makefile

```diff
@@ -29,7 +29,51 @@ bash.done: Makefile systemd.done
 systemd.done: Makefile systemd.src
 	{ [ ! -e systemd_build ] || rm -Rf systemd_build; }
 	mkdir systemd_build
-	cd systemd_build && ../systemd/configure --disable-python-devel --disable-dbus --disable-kmod --disable-blkid --disable-chkconfig --disable-pam --disable-acl --disable-smack --disable-gcrypt --disable-elfutils --disable-libcryptsetup --disable-qrencode --disable-microhttpd --disable-gnutls --disable-binfmt --disable-vconsole --disable-readahead --disable-bootchart --disable-quotacheck --disable-tmpfiles --disable-sysusers --disable-randomseed --disable-backlight --disable-rfkill --disable-logind --disable-machined --disable-timedated --disable-timesyncd --disable-localed --disable-coredump --disable-polkit --disable-resolved --disable-networkd --disable-efi --disable-multi-seat-x --disable-myhostname --disable-gudev --disable-manpages --disable-tests --disable-blkid --disable-hibernate --disable-terminal --disable-hwdb --disable-importd && $(MAKE) && DESTDIR=$${PWD}/installed make install-strip
+	cd systemd_build && ../systemd/configure \
+		--disable-python-devel \
```

Comment:
ah, ty for the info

On Mon, May 11, 2015 at 2:06 PM, Vito Caputo <notifications@github.com>
wrote:

> In stage1/rootfs/usr_from_src/Makefile
> <https://github.com/coreos/rkt/pull/884#discussion_r30082080>:
>
> > @@ -29,7 +29,51 @@ bash.done: Makefile systemd.done
> >  systemd.done: Makefile systemd.src
> >  	{ [ ! -e systemd_build ] || rm -Rf systemd_build; }
> >  	mkdir systemd_build
> > -	cd systemd_build && ../systemd/configure --disable-python-devel --disable-dbus --disable-kmod --disable-blkid --disable-chkconfig --disable-pam --disable-acl --disable-smack --disable-gcrypt --disable-elfutils --disable-libcryptsetup --disable-qrencode --disable-microhttpd --disable-gnutls --disable-binfmt --disable-vconsole --disable-readahead --disable-bootchart --disable-quotacheck --disable-tmpfiles --disable-sysusers --disable-randomseed --disable-backlight --disable-rfkill --disable-logind --disable-machined --disable-timedated --disable-timesyncd --disable-localed --disable-coredump --disable-polkit --disable-resolved --disable-networkd --disable-efi --disable-multi-seat-x --disable-myhostname --disable-gudev --disable-manpages --disable-tests --disable-blkid --disable-hibernate --disable-terminal --disable-hwdb --disable-importd && $(MAKE) && DESTDIR=$${PWD}/installed make install-strip
> > +	cd systemd_build && ../systemd/configure \
> > +		--disable-python-devel \
>
> When I originally made the list of 'disable everything' it was done in the configure
> --help output order.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/884/files#r30082080>.
>